corehq/apps/cloudcare/templates/formplayer/menu_list.html

```diff
@@ -71,7 +71,11 @@ <h1 class="page-title"><%- title %></h1>
         </div>
     </div>
     <% if (menuIndex % 2 === 1) { %>
-    <div class="clearfix"></div>
+      <div class="clearfix visible-xs"></div>
```

Comment:
clearfix forces a clean break in the columns. so if you have a xs screen size, with normal column layout, you get:
```
A B
C D
```
However if the A box has a higher height than the B box (happens when there is more text in the module) then you get:
```
A B
  C
D
```
Which is annoying. the way to fix it, is to add a clear fix element after the first two elements. or after the first 3, or the first 4. if you were to do it after every item, your screen would look like:
```
A
B
C
D
```